src/Config.ts

```diff
@@ -0,0 +1,76 @@
+import { ClientConfig } from "pg";
+
+/**
+ * The configuration for schemalint.
+ */
+type Config = {
```

Comment:
I chose the short name `Config`. Would you prefer a more descriptive name, for example `SchemalintConfig`?

Reply:
I think this is fine. It will likely only be used in configuration files where it will be clear what's being referenced.